NOTICE.txt

```diff
@@ -0,0 +1,83 @@
+React-Spectrum
+=================
+The following is a list of sources from which code was used/modified in this codebase.
+
+-------------------------------------------------------------------------------
+This codebase contains a portion of code from react which can be obtained at:
+  * LICENSE:
```

Comment:
should the link go first here? seems weird to say it "can be obtained at", and then no link.